Source/Fuse.Nodes/Node.uno

```diff
@@ -156,5 +156,22 @@ namespace Fuse
 			}
 			return null;
 		}
+		
+		public T FindByType<T>() where T : class
```

Comment:
Right, I guess that makes sense. So we're not interested in narrowing it down later either, sounds good.